### PR TITLE
Make Sure `accountDetails` are Initially Loaded when `DetailsForm` renders

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4838

state-is-there

### CODE REVIEW

File: components/expenses/PayoutBankInformationForm.js [hunk 1]
```diff
@@ -211,14 +211,20 @@ FieldGroup.propTypes = {
 };
 
 const DetailsForm = ({ disabled, getFieldName, formik, host, currency }) => {
+  const [accountDetails, setAccountDetails] = useState(null);
+  useEffect(() => {
+    const data = get({ ...formik.values }, getFieldName('data'));
+    setAccountDetails(data);
+  }, []);
```
Comment (Betree) [inline components/expenses/PayoutBankInformationForm.js:218]: Why not simply:

```suggestion
  const [accountDetails, setAccountDetails] = useState(() => get(formik.values, getFieldName('data')));
```

**Review by Betree — COMMENTED** [review]
Sorry for the late review here, this PR slipped my attention.

File: components/expenses/PayoutBankInformationForm.js (hunk 1 above)
Comment (SudharakaP) [reply components/expenses/PayoutBankInformationForm.js]: I think we don't need this `useState` at all. I've simplified the code. 😉

**Review by Betree — APPROVED** [review]
I haven't tried to reproduce the issue, but the code looks fine. If you've tested it carefully and are confident that this will resolve https://github.com/opencollective/opencollective/issues/4838, I'm all good to merge :+1:

**Comment by Betree** [comment]
Reverting this in https://github.com/opencollective/opencollective-frontend/pull/7138

**Pull request merged**